src/app/footer-navbar/footer-navbar.component.html

```diff
@@ -1,12 +1,20 @@
-<footer>
-  <div class="left-side">
-      <a routerLink="/about" routerLinkActive="active">About</a>
-      <a href="//blog.fossasia.org">Blogs</a>
-      <a href="//github.com/fossasia/susper.com">Code</a>
+<nav class="navbar navbar-default footer navbar-fixed-bottom" role="navigation">
+  <div class="navbar-header">
+    <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+      <span class="icon-bar"></span>
+      <span class="icon-bar"></span>
```

Comment:
that's for this responsive button
![image](https://cloud.githubusercontent.com/assets/15216503/25774287/2dd828ca-32aa-11e7-89fa-64009ee498d0.png)

on small screens